# 传感器监测物体所有信息

uint32            u4Id
uint32            u4SensorId
uint16            u2Flags
uint8             u1Type
int8              u1Occlusion
int8              u1DeltaLanePos
uint8             u1ValidFlag
float64           u8ExistProb
float64           u8TypeIdProb
object_uion       sExtraInfo